Sources/Public/DotLottie/Utility/DotLottieAnimation.swift

```diff
@@ -0,0 +1,52 @@
+//
+// DotLottieAnimation.swift
+// Pods
+//
+// Created by Evandro Harrison Hoffmann on 28/06/2021.
+//
+
+import Foundation
+
+struct DotLottieAnimation: Codable {
```

Comment:
These files with only `internal` declarations should be in `Sources/Private/Model/DotLottie`